Sources_v3/StreamChatUI/ChatChannel/Reactions/ChatMessageReactionsBubbleTail.swift

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+enum ChatMessageReactionsBubbleTail {
+    struct Options {
+        let smallRadius: CGFloat
+        let bigRadius: CGFloat
+        let border: CGFloat
+        let outline: CGFloat
+        let flipped: Bool
+
+        static func small(flipped: Bool) -> Self {
+            .init(
+                smallRadius: 2,
+                bigRadius: 4,
+                border: 1,
+                outline: 2,
+                flipped: flipped
+            )
+        }
+
+        static func large(flipped: Bool) -> Self {
+            return .init(
+                smallRadius: 3,
+                bigRadius: 6,
+                border: 0,
+                outline: 1,
```

Comment:
Large tail does not have outline

Reply:
The outline is needed here to share the same drawing logic as for small tails.  It's just it has `.clear` color and visually looks like a distance in-between small and big bubbles.